users/config.rst

```diff
@@ -98,13 +99,14 @@ The following shows the default configuration file:
             <restartOnWakeup>true</restartOnWakeup>
             <autoUpgradeIntervalH>12</autoUpgradeIntervalH>
             <keepTemporariesH>24</keepTemporariesH>
-            <cacheIgnoredFiles>true</cacheIgnoredFiles>
+            <cacheIgnoredFiles>false</cacheIgnoredFiles>
             <progressUpdateIntervalS>5</progressUpdateIntervalS>
             <symlinksEnabled>true</symlinksEnabled>
             <limitBandwidthInLan>false</limitBandwidthInLan>
-            <databaseBlockCacheMiB>0</databaseBlockCacheMiB>
             <minHomeDiskFreePct>1</minHomeDiskFreePct>
             <releasesURL>https://api.github.com/repos/syncthing/syncthing/releases?per_page=30</releasesURL>
+            <overwriteNames>false</overwriteNames>
```

Comment:
Bit late in the day for that. Maybe we should delay the v0.13 release for another week and take some time to figure this and the proper solution to the other PR out